### ISSUE TITLE
style not defined

### ISSUE THREAD

**Issue by jclsn** [JuliaPlots/PlotlyJS.jl#475]
**Describe the bug**
I am not able to apply any styles as mentioned in the documentation. It seems that neither the ``use_styles!()`` function or style parameter in the ``plot()`` function are defined. What am I doing wrong?

```
julia> MK2SideAPlot = plot([tr_a, tr_b], layout, style=style(:ggplot))
ERROR: UndefVarError: `style` not defined
Stacktrace:
 [1] top-level scope
   @ REPL[208]:1
```

**Version info**

Please provide the following:

1. Julia version: 1.9.3
2. Output running the following in Julia 0.7 or greater: `using Pkg; pkg"status"` (if you are on Julia 0.6 or earlier run `Pkg.status()`)
```
julia> using Pkg; pkg"status"
Status `~/.julia/environments/v1.9/Project.toml`
  [ccf2f8ad] PlotThemes v3.1.0
  [f0f68f2c] PlotlyJS v0.18.11
  [91a5bcdd] Plots v1.39.0
  [8149f6b0] WAV v1.2.0
```

**Comment by empet** [comment]
Starting with the PlotlyJS.jl v0.18.0 the style has been removed, because  its  functionality  is now  available in plotly templates.
See this closed issue [https://github.com/JuliaPlots/PlotlyJS.jl/issues/408](https://github.com/JuliaPlots/PlotlyJS.jl/issues/408).

**Comment by jclsn** [comment]
@empet Maybe update the documentation accordingly then?

**Comment by empet** [comment]
Here are the docs for the last version: [https://github.com/plotly/plotlyjs.jl-docs/tree/master/julia](https://github.com/plotly/plotlyjs.jl-docs/tree/master/julia) or [https://plotly.com/julia/](https://plotly.com/julia/), and here the  trace and layout  reference [https://plotly.com/julia/reference/](https://plotly.com/julia/reference/).

**Comment by jd-foster** [comment]
Thanks for the prompt. I've just come on as a new maintainer and become aware of this last week. The docs will be updated soon.

**Comment by juliohm** [comment]
The documentation is still outdated, mentioning Styles and style everywhere. Can a maintainer please erase these sections from the docs?

**Comment by jd-foster** [comment]
Thank you for the reminder.

**Comment by RoyiAvital** [comment]
I encountered the same issue when trying to access `PlotlyJS.STYLES` which is no longer available yet mentioned on documentation.

**Comment by mzaffalon** [comment]
@jd-foster can this issue be closed?

**Comment by jd-foster** [comment]
Yes, and anyone interested in contributing a section of documentation on templates would be welcome.
See: https://github.com/JuliaPlots/PlotlyJS.jl/issues/519#issuecomment-4160090203
